### PR TITLE
[docker] fixes #1577 - Install valgrind package inside skycoin/skycoindev-cli image

### LINKED ISSUE #1577: Add valgrind in skycoin/skycoin-dev image
... to track memory leaks , useful for detecting issues in apps built with `libskycoin` .

### PR DESCRIPTION
**Pull request by stdevNorge**
Fixes #1577 

Changes:
- Add valgrind package inside skycoin/skycoindev-cli image

Does this change need to mentioned in CHANGELOG.md?
Yes

### CODE REVIEW

File: docker/images/dev-cli/Dockerfile [hunk 1]
```diff
@@ -135,6 +136,9 @@ RUN set -ex ; \
     screen \
     sudo \
     doxygen \
+    valgrind \
+    bsdmainutils \
+    swig \
```
Comment (olemis) [inline docker/images/dev-cli/Dockerfile]: Remove this . Installed from sources since #1667 .

Comment (olemis) [reply docker/images/dev-cli/Dockerfile]: maybe just merge with `develop` branch ?

**Comment by olemis** [comment]
@gz-c ready

**Comment by gz-c** [comment]
conflict again

**Comment by olemis** [comment]
@gz-c updated once again ... thnx for the pointer

**Pull request merged**